### ISSUE TITLE
Collect examples from various alignment tools

### ISSUE THREAD

**Issue by menzowindhouwer** [CLARIAH/clariah-plus#76]
We need example linksets from the alignment tools in CLARIAH:

- WP2: LL
- WP4: BurgerLinker
- WP5: Cultuurlink
- WP5: Amalgame

**Comment by wmelder** [comment]
WP5: cultuurlink.beeldengeluid.nl has been taken down. Presumably, the reason is lack of uptake in the cultural heritage community. So, SKOS alignments, for now, can not be made using cultuurlink. No example can be delivered.

**Comment by wxwilcke** [comment]
BurgerLinker produces a new dataset, rather than a linkset, in which all people it deems the same person are collapsed into a single entity with a new URI. Each of these entities has one or more 'civ:personID' properties, which specify the identifiers of the different instances of the same person in the original (CSV) dataset.

**Comment by ddeboer** [comment]
Do you by ‘alignment’ mean ‘reconciliation’? In that case, it’s relevant to note that [NDE’s Network of Terms now supports the Reconciliation Service API](https://github.com/netwerk-digitaal-erfgoed/network-of-terms/tree/master/packages/network-of-terms-reconciliation). We’ve started with support for the RKDartists and CHT terminology sources, and will soon be adding all other sources in the Network of Terms (in any case, those sources that don’t offer a Reconciliation Service themselves).

**Comment by wmelder** [comment]
@ddeboer No, an alignment is a mapping between two vocabularies. A reconciliation is actually a replacement of local values in a dataset with URI's and labels from a authorative source.

**Comment by wmelder** [comment]
Amalgame examples can be found in [Victor's reply](https://github.com/CLARIAH/clariah-plus/issues/89#issuecomment-1049920691)

**Comment by ddeboer** [comment]
Ah, I was confused by 

> Alignment/matching (reconciliation)

in https://github.com/CLARIAH/clariah-plus/blob/ce4aee637fb0e8e2539a126ceda228c7303c3291/interest-groups/vocabularies/README.md

**Comment by wmelder** [comment]
@ddeboer I proposed this [service](https://docs.google.com/document/d/1dCTK5w9jJRKIQuQ9t_xl7YbTtFljLoLTNT3C2EEIPtg/edit#heading=h.dy1j2chjicwy) to the SDR, but it didn't make it. But we do have the FAIR vocabularies service and maybe reconciliation can be part of it.

**Comment by roelandordelman** [comment]
@wmelder re the reconciliation service that didn't make it: it was not included as a task within the SDR as it was not clear where to put it. If it can be part of FAIR vocab that would be great.
